resources/assets/coffee/react/beatmap-discussions/discussion.coffee

```diff
@@ -117,7 +117,7 @@ class BeatmapDiscussions.Discussion extends React.PureComponent
 
     topClasses = "#{vbn} #{vbn}--#{type}"
     topClasses += " #{vbn}--inactive" if score != 0
-    disabled = @isOwner() || (type == 'down' && !@canDownvote()) || @props.currentBeatmap.deleted_at?
+    disabled = @isOwner() || (type == 'down' && !@canDownvote()) || (@props.discussion.beatmap_id? && @props.currentBeatmap.deleted_at?)
```

Comment:
the check for `beatmap_id` and `deleted_at` look like they should be consolidated together (same as the change above)